odc/stats/io.py

```diff
@@ -400,7 +400,7 @@ def dump_with_eodatasets3(
         sha1_url = task.metadata_path("absolute", ext="sha1")
         proc_info_url = task.metadata_path("absolute", ext=self._proc_info_ext)
         dataset_assembler = task.render_assembler_metadata(
-            ext=self._band_ext, output_dataset=ds
+            ext=self._band_ext, output_dataset=ds, use_center_time=hasattr(proc, "CENTER_TIMERANGE")
```

Comment:
Will setting `CENTER_TIMERANGE = False` in a class work as intended with this logic?

Reply:
I have to admit I would be a bit surprised if I set `CENTER_TIMERANGE = False` in a class and that gave me center time.

I'm not the target user for this though, so that probably doesn't matter. But how are classes that don't want center time but inherit from a class that sets `CENTER_TIMERANGE` meant to deal with their situation?